test/TestsHelper.js

```diff
@@ -554,6 +555,13 @@ TestsHelper.execCmdWithAssertion = function (cliCmd, cmdOptions, apiOptions, sna
         // ensure line separators are always the same
         finalOutput = finalOutput.replace(/\r\n/g, '\n');
 
+        // replace a given text/regex if an object with 'oldValue' and 'newValue' fields is submitted.
+        if (replacementObject && typeof replacementObject === 'object') {
+          const matchedText = finalOutput.match(replacementObject.oldValue)[0];
+          expect(matchedText).to.exist;
```

Comment:
There's no need to assert that the text is found. If it isn't found, the snapshot will be different.